rpc/src/v1/tests/mocked/secretstore.rs

```diff
@@ -100,7 +101,29 @@ fn rpc_secretstore_shadow_decrypt() {
 }
 
 #[test]
-fn rpc_secretstore_sign_servers_set() {
+fn rpc_secretstore_servers_set_hash() {
+	let deps = Dependencies::new();
```

Comment:
You can consider using new `jsonrpc_test`:
https://github.com/paritytech/jsonrpc/blob/master/test/src/lib.rs#L28